dev-perl/Array-Diff/Array-Diff-0.70.0-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DIST_AUTHOR=TYPESTER
+DIST_VERSION="0.07"
+
+inherit perl-module
+
+DESCRIPTION="Find the differences between two arrays"
+LICENSE="|| ( Artistic GPL-2 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test +xs"
+COMMON_DEPEND="
+	>=dev-perl/Algorithm-Diff-1.190.0
+	dev-perl/Class-Accessor
+	xs? (
+		dev-perl/Algorithm-Diff-XS
+	)
+"
+DEPEND="
+	${COMMON_DEPEND}
+	dev-perl/Module-Install
```

Comment:
Build-Time dependency on `dev-perl/Module-Install` is likely an error, packages that use `Module::Install` should be bundling `Module::Install` in `inc/` and depending on it externally is either harmful or useless. (Plans are to try removing it from ::gentoo entirely when feasible)